scripts/create-cert.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+openssl req -nodes -new -x509 -keyout test/fixtures/fake-key.pem -out test/fixtures/fake-cert.pem -subj '/CN=US'
```

Comment:
Can we add a comment here for what the use case is here and when we'd need to re-run it (I see it's in scripts already which is good)

Reply:
added a comment :+1: